shopno/src/styles/popUp.scss

```diff
@@ -0,0 +1,25 @@
+#examplesPopup {
+    position: fixed;
+    z-index: 100;
+    top: 0;
+    left: 0;
+    display: none;
+    justify-content: center;
+    align-items: center;
+    width: 100vw;
+    height: 100vh;
+    background: #686608a1;
```

Comment:
use css variables

Reply:
This color is used once

Reply:
ok, but I can find places where you can use the same color at least twice and you don't use css variables. You should also understand that's not about code repetition. It also useful for other developers to read your code just by looking on semantically correct names.